assets/dev/js/frontend/handlers/base.js

```diff
@@ -29,6 +29,47 @@ module.exports = elementorModules.ViewModule.extend( {
 		return true;
 	},
 
+	isOverflowHidden() {
+		return 'hidden' === this.$element.css( 'overflow' );
+	},
+
+	getOffset() {
+		if ( 'body' === elementor.config.document.container ) {
+			return this.$element.offset().top;
+		}
+
+		const $container = jQuery( elementor.config.document.container );
+		return this.$element.offset().top - $container.offset().top;
+	},
+
+	setHandlesPosition( insideHandleClass ) {
+		const document = elementor.documents.getCurrent();
+
+		if ( ! document || ! document.container.isEditable() ) {
+			return;
+		}
+
+		if ( 'elementor-section--handles-inside' === insideHandleClass && elementor.settings.page.model.attributes.scroll_snap ) {
```

Comment:
I see that this string `elementor-section--handles-inside` appears several times.
Maybe we can set it as a const so we will have one source of truth for it?